Marlin/utility.cpp

```diff
@@ -21,9 +21,10 @@
  */
 
 #include "Marlin.h"
```

Comment:
This line can be removed entirely, since the only dependency is actually "temperature.h" (and it can decide what other dependencies to apply.)